cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -474,20 +483,19 @@ void ThinClientPoolDM::cleanStaleConnections(std::atomic<bool>& isRunning) {
                 nextIdle < _nextIdle) {
               _nextIdle = nextIdle;
             }
-            savelist.push_back(conn);
+            put(conn, false);
           }
         }
       }
       replaceCount--;
+      count++;
```

Comment:
I'd be more comfortable with `std::this_thread::yield` every time through this loop. Let the scheduler decide if and when there are free cycles to run this task hot or not. Remove the counter and mod condition.

Reply:
Testing of solution alternatives is ongoing. Ticket will be updated with test results.

Reply:
After testing of yield solution, it is observed that delays are same as without solution, which is not acceptable. Delays are in range of 1 - 2 seconds per get at some peeks.
When tested solution with "sleep", max delays are cca. 150 ms.

Reply:
Odd... Ok.